db/migrate/20230203165245_create_viewing_parties.rb

```diff
@@ -0,0 +1,13 @@
+class CreateViewingParties < ActiveRecord::Migration[5.2]
+  def change
+    create_table :viewing_parties do |t|
+      t.integer :movie_id
+      t.references :user, foreign_key: true
+      t.date :day
+      t.time :start_time
+      t.integer :duration
+
```

Comment:
Definitely a good idea to have ViewingParties save some movie attributes so that we don't have to keep making movie api calls